spec/git_reflow/all-approval/git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
+    # Stubbing out numlgtm value to test all reviewers in gitconfig file
+    GitReflow::Config.stub(:get).with("constants.numlgtm").and_return('')
```

Comment:
There are quite a lot of the these stubs, for the purposes of this commit, let's leave this clean up task for another day. I would like to merge this change in for now. I think a regex expression would clean this up quite well 👍